api/chat.go

```diff
@@ -573,8 +574,7 @@ func CollectStats(daoWrapper dao.DaoWrapper) func() {
 			}
 			if s, err := daoWrapper.GetStreamByID(context.Background(), fmt.Sprintf("%d", sID)); err == nil {
 				if s.LiveNow { // store stats for livestreams only
-					s.Stats = append(s.Stats, stat)
-					if err := daoWrapper.AddStat(stat); err != nil {
+					if err := stats.Client.AddStreamStat(fmt.Sprintf("%d", s.CourseID), stat); err != nil {
```

Comment:
Here the live viewers are counted. Can we also add a point for every vod view? (https://github.com/joschahenningsen/TUM-Live/blob/5aa33d394ed3b639e703fcdfc9eac4521bee5bd6/api/chat.go#L634)

Reply:
Also the `AddVodView` dao stuff can then be removed.